core/shared/src/main/scala/com/monovore/decline/Theme.scala

```diff
@@ -0,0 +1,50 @@
+package com.monovore.decline
+
+import cats.Show
+import cats.data.NonEmptyList
+import cats.syntax.all._
+import com.monovore.decline.HelpFormat.Plain
+import com.monovore.decline.HelpFormat.Colors
+
+private[decline] trait Theme {
```

Comment:
I don't follow why we need both `Theme` and `HelpFormat` - seems like users could pass in a `Theme` directly. (And if we can get away with fewer concepts, strong preference for that!)

Reply:
> seems like users could pass in a Theme directly

I treat `Theme` interface as almost an implementation details - the methods in it are very tightly coupled to the help rendering logic (they're by no means the minimal set of methods required to render something), and I don't think it's beneficial (at **this point**) to expose this interface to users at all - because it'd increase compatibility surface.

Alternatively I could make it all 1 concept and make every method in Theme private to address the concerns from previous paragraph - but I felt like keeping Theme separate and private is the easiest w.r.t. API surface and flexibility.

Reply:
Yeah, it's true that the fact that it's not user-facing does mitigate it to a large extent. Willing to keep it as is for now...